enhancements/opm-package-filtering.md

```diff
@@ -0,0 +1,79 @@
+---
+title: opm-package-filtering
+authors:
+  - "@djzager"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-04-03
+last-updated: 2020-04-03
+status: implementable
+---
+
+# OPM Package Filtering
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+Add support to the `opm` command line utility to filter operators from an
+operator index (or registry). This enables consumers of operator indexes to take
+an existing, and presumably large, index and generate a new index with only the
+operators they wish to make available in the catalog. In short, this is the
+inverse of `opm (index|registry) rm` where you select which operators to remove,
+`opm (index|registry) filter` allows you to select the operators to keep.
+
+## Motivation
+
+As Operator adoption expands and OLM usage increases the index images used to
+make operator catalogs available to OLM will grow. Cluster administrators
+__should__ have a way to curate operator catalogs. This feature provides a basic
+mechanism for catalog curation.
+
+### Goals
+
+- Allow `opm` user to select operators to keep from an existing registry or
+  index image.
+
+### Non-Goals
+
+- Manipulate the available channels of any operator in the registry or index
+  image.
+
+## Proposal
+
+### Filter Subcommand
+
+Adding the `filter` subcommand to the `index` and `registry` command provides a
```

Comment:
nit: `prune`?

Reply:
looking at the definition of `prune`. I think you may be right that it is a better fit.